tests/pages/api/annotationAPI.js

```diff
@@ -0,0 +1,34 @@
+const { I } = inject();
+const assert = require('assert');
+
+module.exports = {
+  async setAnnotation(annotationName, tags, nodeName, serviceNames) {
+    const headers = { Authorization: `Basic ${await I.getAuth()}` };
+    const body = {
+      text: annotationName,
+      tags: [tags],
+      node_name: nodeName,
+      service_names: [serviceNames],
+    };
+
+    const resp = await I.sendPostRequest('v1/management/Annotations/Add', body, headers);
+
+    return resp.status;
```

Comment:
We need an assert here

Reply:
@yurkovychv When I will add assert here, than I will not be able to return status for the tests when I am checking the status. Or I can add there again: `/* eslint-disable consistent-return */` and then I can have assert and return in if condition for the errors. What do you think? :)

Reply:
what about an additional parameter in a function that will accept a status code to assert?

Reply:
Ok.